src/core/storage.tf

```diff
@@ -0,0 +1,40 @@
+resource "azurerm_resource_group" "rg_storage" {
+  name     = format("%s-storage-rg", local.project)
+  location = var.location
+  tags     = var.tags
+}
+
+## Storage account to save blob
+#tfsec:ignore:azure-storage-default-action-deny
+module "selc-blob-storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s-blobstorage", local.project), "-", "")
+  account_kind               = "StorageV2"
+  account_tier               = "Standard"
+  account_replication_type   = "LRS"
```

Comment:
since in this storage will be stored contracts I suggest to use RA-GZRS for production env (LRS for dev,uat)
https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy#durability-and-availability-parameters

Reply:
@iwoak how much days should we configure the delete retention period in prod? in other environment it'sok to set 1 day?